scripts/partition_manager.py

```diff
@@ -963,26 +967,36 @@ def test():
         # Gap from deleted partition.
     }
 
-    start, size = get_dynamic_area_start_and_size(test_config, 100, 'app')
+    start, size = get_dynamic_area_start_and_size(test_config, 0, 100, 'app')
     assert start == 20
     assert size == 80
 
     test_config = {
         'app':    {'address': 0,    'size': 10},
         # Gap from deleted partition.
         'second': {'address': 40, 'size': 60}}
-    start, size = get_dynamic_area_start_and_size(test_config, 100, 'app')
+    start, size = get_dynamic_area_start_and_size(test_config, 0, 100, 'app')
     assert start == 0
     assert size == 40
 
     test_config = {
         'first': {'address': 0,    'size': 10},
         # Gap from deleted partition.
         'app':   {'address': 20, 'size': 10}}
-    start, size = get_dynamic_area_start_and_size(test_config, 100, 'app')
+    start, size = get_dynamic_area_start_and_size(test_config, 0, 100, 'app')
     assert start == 10
     assert size == 100 - 10
 
+    # Verify that base address can be larger than size (RAM vs FLASH)
```

Comment:
Just a nit, but AFAIU the error, it's from the fact that RAM's base address is something other than 0.
```suggestion
    # Verify that base address can be larger than 0 (RAM vs FLASH)
```

Reply:
Will fix if CI is red and I need to force push :D